docs/decisions/0010-taxonomy-enable-context.rst

```diff
@@ -0,0 +1,56 @@
+10. Taxonomy enabled for context
+================================
+
+Context
+-------
+
+The MVP specification says that taxonomies need to be able to be enabled/disabled for the following contexts: instance, organization, and course.
+
+Content Authors must be able to turn taxonomies (instance and org-levels) on/off at the course level.
+
+Decision
+--------
+
+Instance
+~~~~~~~~
+
+A Taxonomy can be enabled/disabled for all contexts using its ``enabled`` flag.
```

Comment:
I think this needs to be clarified. It sounds like this is saying "the taxonomy can be enabled for all contexts (forced on for all courses, regardless of other settings) when `enabled=True` or disabled for all contexts (forced off for all courses, regardless of other settings) when `enabled=False`". We either need a third state (can be selectively enabled/disabled at other levels when this is set to `None`), or make this into either a "global disable" or "global enable" boolean, but not both. I hope that makes sense.

Reply:
Yep, that makes sense @bradenmacdonald . Does this look clearer? https://github.com/open-craft/openedx-learning/blob/jill/tagging-data-arch-adr/docs/decisions/0010-taxonomy-enable-context.rst